src/main/java/com/teamacronymcoders/epos/locks/keys/harvest/ToolHarvestLockKey.java

```diff
@@ -0,0 +1,110 @@
+package com.teamacronymcoders.epos.locks.keys.harvest;
+
+import com.teamacronymcoders.epos.api.locks.keys.GenericLockKey;
+import com.teamacronymcoders.epos.api.locks.keys.IFuzzyLockKey;
+import com.teamacronymcoders.epos.api.locks.keys.ILockKey;
+import com.teamacronymcoders.epos.locks.FuzzyLockKeyTypes;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.ToolType;
+
+//TODO: Add back in some form of support based on tool type
```

Comment:
I will make sure to cleanup this class a bit more and implement things that are more tool type specific before we end up merging.